graphql/schemabuilder/reflect.go

```diff
@@ -107,6 +107,56 @@ var scalarArgParsers = map[reflect.Type]*argParser{
 			return nil
 		},
 	},
+	reflect.TypeOf(int8(0)): {
```

Comment:
```
--- FAIL: TestArgParser (0.00s)
	reflect_test.go:571: expected unsupported fields to fail
```

it looks like we had a test to verify that these were not supported, so that'll need a fixup. 😅